### ISSUE TITLE
Problems: filter action should have aria-expanded set

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#97125]
Environment Details:
VSCode Version : 1.44.2
OS Version : Win10
Additional Details:
Screenreader: NVDA[2019.3.1], JAWS[2020.70]
GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode; #MAS1.3.1; #MAS4.2.1

**Repro Steps:**
1)Open Visual studio code editor  
2)Press ctrl+shift+M
3)Validate it's possible to navigate between items in the problems view
4)Navigate to the Problem tab and then navigate to Filter button.
5)Open Jaws/NVDA and check whether it reads collapsed/Expanded after click enter on the filter button.
**Actual:**
Screen reader does not announce the state as Collapsed/Expanded when click enter on filter button.

**Expected:**
Screen reader should announce the state as Collapsed/Expanded when click enter on filter button.

**User Impact:**
Screen reader will not able to understand whether state of the scroll lock button is lock or unlocked

**Attachment For Reference:**

[97125_A11y_VSCode_Problems Panel_ScreenReaderScreen reader does not announce the state as CollapsedExpanded when click enter on filter button.zip](https://github.com/microsoft/vscode/files/4591622/97125_A11y_VSCode_Problems.Panel_ScreenReaderScreen.reader.does.not.announce.the.state.as.CollapsedExpanded.when.click.enter.on.filter.button.zip)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@nirmalsahoo good issue. Agree with this. 
@sandy081 can you just set `aria-expanded: false` on the "More Filters..."  button that way users will be aware that they can expand it to get the menu.

**Comment by sandy081** [comment]
Does the value of `aria-expanded` to be fixed to `false` or change according to the menu?

**Comment by isidorn** [comment]
You can change according to menu.
But in practice that will not happen -> since you can not have focus on tha action and have the menu open.

**Comment by sandy081** [comment]
@isidorn Does this attribute has to be set on all drop down triggering actions?

https://github.com/microsoft/vscode/blob/ff118340b561fea6184172ffaf3bbf859b21055b/src/vs/base/browser/ui/dropdown/dropdown.ts#L305-L305

**Comment by isidorn** [comment]
@sandy081 yes, that would be great.
Ideally we would set the aria-expanded value according to the menu, since maybe some action in the future will allow to have the menu open and focus somewhere else.

**Comment by isidorn** [comment]
@sandy081 thanks for fixing this! I just tried it out from source and it looks good, thus adding `verified` label.
